### DISCUSSION TITLE
Web Player

### DISCUSSION THREAD

**Discussion by Tiptoemovie071** [ciderapp/Cider#1406] [Feature Request]
I would love to be able to listen to music on the web! I understand if this doesn’t work out I understand. I’m also fairly new to Apple music (after switching from Spotify which has this feature!)

**Comment by tva2000hn** [comment]
Hmm the Apple Music web player should got you covered: https://music.apple.com

**Reply by AW1534** [reply]
That's right, but apple's auth system is a pain. forgive me if i'm wrong, but since It's an electron app, wouldn't it be as easy as just copying the html, css, frontend js code?
